packages/shared/package.json

```
@@ -0,0 +1,22 @@
+{
+	"name": "shared",
+	"version": "1.0.0",
+	"type": "module",
+	"main": "build/index.js",
+	"types": "build/index.d.ts",
+	"engines": {
+		"node": "18.x.x",
+		"npm": "10.x.x"
+	},
+	"scripts": {
+		"postinstall": "npm run build",
+		"lint:type": "npx tsc --noEmit",
+		"lint:js": "npx eslint . --max-warnings=0 --no-warn-ignored",
+		"lint": "concurrently \"npm:lint:*\"",
+		"build": "tsc && npm run build:copy",
```

since you have `tsconfig.build.json` you need to provide it in all build scripts (for all packages/apps), because `tsconfig.json` is taken by default
```suggestion
		"build": "tsc --project tsconfig.build.json  && npm run build:copy",
```

https://github.com/what1s1ove/whatislove.dev/blob/main/packages/shared/package.json#L11